src/component/playground/Playground/playground.model.ts

```diff
@@ -0,0 +1,248 @@
+export enum PlaygroundFeatureSchemaVariantPayloadTypeEnum {
```

Comment:
Is this file auto-generated? seems like it should be.

Reply:
I extracted the Playground models from the generated files